team-13.md

```diff
@@ -0,0 +1,148 @@
+# Chrome Extension Idea: TypeMaster (tentative)
+
+## Authors
+
+Tiffany Lian, Alexander Dempsey, James Huang, Sanya Shetty
+
+## Problem Statement
+
+Many application portals require users to meet word count limits which can be both challenging and demotivating. Without any built-in word count features on the portal users would have to resort to external tools like pasting into Google Docs which is inefficient. Moreover, integrating AI-powered semantic analysis and real-time analytics into the user interface could streamline the process, providing insightful feedback and enhancing productivity and user experience.
+
+## Target Audience
+
+- The target audience for WordCounter includes:
+
+- Students working on assignments with word requirements set by their instructors.
+
+- Editors and Proofreaders who want to estimate the time required for a task.
+
+- Applicants applying to jobs, clubs, or colleges who have prompts with word requirements.
+
+
+## Description
+
+TypeMaster is a portable writing assistant that provides word counter, writing analytics, and helps users visualize their writing goals on any website, 
+
+## Selling Points
+
+1. Allows uninterrupted workflow, accessible from within the tab.
+2. Achieve writing milestones effortlessly by setting customizable word count goals and receiving visual prompts as you progress.
+3. Persistent settings across browsing sessions and websites
+4. Maximize productivity with quick and easy access.
+5. Accurate word counts and analytics displayed in a compact, unobtrusive manner.
+6. OpenAI API integration which provides you feedback on your professional applications.
+
+## User Stories
```

Comment:
The user stories do a great job at outlining how the main value proposition (the word counter and the analytics) would be used overall. This would become easier to detail once you start working on the UI, but I wonder if you could make the user stories more specific to the features of the product (eg. the highlight to view word count or right click to check tone of the text) that the user would interact with, rather than the value proposition (providing the word count or the use of analysitc).